pm/ticket.go

```diff
@@ -14,6 +14,16 @@ const (
 	bytes32Size = 32
 )
 
+type TicketParams struct {
```

Comment:
I've been trying to add comments for all exported functions/data (i.e. capitalized variables) so that godoc can then generate nice documentation for everything (see: https://golang.org/doc/effective_go.html#commentary) so I thought we can make this an ongoing practice - thoughts?

Reply:
[golint](https://github.com/golang/lint) seems to do the trick. I use VSCode which runs golint automatically in the editor. 

On the topic of linters, there seems to be a fairly large amount of static analysis/linting tools for Go mentioned in the README for [gometalinter](https://github.com/alecthomas/gometalinter) which looks like a tool that can run a bunch of them simultaneously. We're probably fine with just adding golint to the CI process initially (though we'll have to fix a bunch of its warnings around things like missing godoc), but perhaps we can explore using gometalinter later on.